sdk/inkrecognizer/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "@azure/inkrecognizer",
```

Comment:
@bterlson , wonder if you can comment on this?

Reply:
I'm not sure what "sdk-type" is used for. @KarishmaGhiya can you chime in?

Reply:
Let me add anyway: if not to be used in Track 2, the parameter will be ignored, but no harm.

Reply:
Also now that I think about it, should this be @azure/ink-recognizer? (It should be if your .net namespace is something like InkRecognizer rather than say Inkrecognizer).

Reply:
Considering there are existing packages in the @azure scope, let's instead align with the other cognitive services packages:

`@azure/cognitiveservices-inkrecognizer`.